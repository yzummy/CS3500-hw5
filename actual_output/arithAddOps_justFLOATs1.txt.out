
___Entering new scope...

TOKEN: FLOATCONST 		 LEXEME: 10.5
CONST -> FLOATCONST
FACTOR -> CONST
TOKEN: ADD 		 LEXEME: +
MULT_OP_LIST -> epsilon
TERM -> FACTOR MULT_OP_LIST
ADD_OP -> +
TOKEN: FLOATCONST 		 LEXEME: 3.14
CONST -> FLOATCONST
FACTOR -> CONST
MULT_OP_LIST -> epsilon
TERM -> FACTOR MULT_OP_LIST
ADD_OP_LIST -> epsilon
ADD_OP_LIST -> ADD_OP TERM ADD_OP_LIST
SIMPLE_ARITHLOGIC -> TERM ADD_OP_LIST
ARITHLOGIC_EXPR -> SIMPLE_ARITHLOGIC
EXPR -> ARITHLOGIC_EXPR
START -> EXPR

---- Completed parsing ----

 Value of the expression is: 21.00

___Exiting scope...

