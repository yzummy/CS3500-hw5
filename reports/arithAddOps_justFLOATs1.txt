
---- Completed parsing ----					---- Completed parsing ----

 Value of the expression is: 21.00			      |	Value of the expression is:  13.64
