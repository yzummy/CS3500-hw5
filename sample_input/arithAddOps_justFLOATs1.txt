10.5 + 3.14
