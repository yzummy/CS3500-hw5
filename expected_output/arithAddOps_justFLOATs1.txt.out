
---- Completed parsing ----

Value of the expression is:  13.64
